deployment/docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: '3.8'
+
+services:
+  cordra:
```

Comment:
I thought you pushed a image to our ecr which we can use

Reply:
Yes forgot to update the docker-compose file, updated it in the next commit